client/lib/application/src/application-lifecycle/components/Loading.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import ThreeColumns from 'common/src/components/pure/ThreeColumns.jsx';
+import Icon from 'react-fa';
+
+const Loading = () => {
+    return (
+        <ThreeColumns
```

Comment:
All components under application-lifecycle are not for reuse (those reside in common) but merely only to split up the application lifecycle main component into smaller parts